app/models/concerns/issued_at.rb

```diff
@@ -16,4 +16,10 @@ module IssuedAt
   def initialize_issued_at
     self.issued_at ||= created_at
   end
+
+  def issued_at_cannot_be_before_2000
+    if issued_at.present? && issued_at < Date.new(2000,1,1)
+      errors.add(:issued_at, "Cannot be before 2000")
+    end
```

Comment:
Is there a reason we can't automatically validate this in the module instead of having to change all the including models to manually call it?

```ruby
included do
  validate :issued_at_cannot_be_before_2000
  ...
end
```

Reply:
Mainly that I hadn't clued in that I could do that?    I'll double-check that there are no other things using the issued_at, but that seems seemly to me.

Reply:
Addressed